### PR TITLE
Add Y block

### LINKED ISSUE #9: Add Y block
## User story

```
As a player,
I want to see the "Y" block appearing during the game,
So that I can fill an Y-like gap with it.
```
## Acceptance criteria

1. The block should be presented on the playfield as:
   ```
   # #
    #
    # 
   ```
1. The block is served along with other blocks during the game.

### PR DESCRIPTION
**Pull request by jsayev**
- Add YBlock class;
- Add javadocs for YBlock;
- Add tests for YBlock class;
- Update package version from 0.7 to 0.8;

Closes #9

### CODE REVIEW

File: src/test/java/com/epam/prejap/teatrees/block/YBlockTest.java [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+        YBlock yBlock = new YBlock();
+        int expectedValue = 3;
+        // when
+        int rowSize = yBlock.rows();
+        // then
+        assertEquals(rowSize, expectedValue);
+    }
+
+    @Test(dataProvider = "provideCoordinatesOfValuesForYShape")
+    public void shouldHaveExpectedValuesAtCoordinatesFromDP(int rowIndex, int colIndex, int expectedValue) {
```
Comment (marcin-ogorzalek-EPAM) [inline src/test/java/com/epam/prejap/teatrees/block/YBlockTest.java:46]:
```suggestion
    public void shouldHaveExpectedValuesAtGivenCoordinates(int rowIndex, int colIndex, int expectedValue) {
```

**Review by marcin-ogorzalek-EPAM — CHANGES_REQUESTED** [review]
Remember to sync with the main repo. It is close to being merged I think

File: src/test/java/com/epam/prejap/teatrees/block/YBlockTest.java (hunk 1 above)
Comment (jsayev) [reply src/test/java/com/epam/prejap/teatrees/block/YBlockTest.java]: Done

File: pom.xml [hunk 2]
```diff
@@ -5,7 +5,7 @@
 
     <groupId>com.epam.prejap</groupId>
     <artifactId>tea-trees</artifactId>
-    <version>0.9</version>
```
Comment (jsayev) [inline pom.xml]: actually 0.9 is my version prior to merge (which brings 0.10) and my change updates it to 0.11

**Review by marcin-ogorzalek-EPAM — APPROVED** [review]

**Pull request merged**